spark/src/test/java/org/opensearch/sql/spark/functions/response/DefaultSparkSqlFunctionResponseHandleTest.java

```diff
@@ -45,7 +45,7 @@ public void testConstruct() throws Exception {
     assertEquals(new ExprByteValue(5), row.get("col5"));
     assertEquals(new ExprDoubleValue(6.1), row.get("col6"));
     assertEquals(new ExprFloatValue(7.1), row.get("col7"));
-    assertEquals(new ExprTimestampValue("2024-01-02 03:04:05.1234"), row.get("col8"));
+    assertEquals(new ExprStringValue("2024-01-02 03:04:05.1234"), row.get("col8"));
```

Comment:
curious if we actually want `ExprDatetimeValue` here instead?

Reply:
The actual representation of the value is String. (sparkType:"timestamp" is considered as ExprCoreType.DATE, and ExprCoreType.DATE item is converted to ExprStringValue. I'm planning to remove this data conversion and directly return spark data types as response.